src/providers/data_provider_fo.c

```diff
@@ -594,6 +594,14 @@ static void be_resolve_server_done(struct tevent_req *subreq)
     tevent_req_error(req, ret);
 }
 
+void dump_be_svc_data(struct be_svc_data *svc)
```

Comment:
This could be `static`?

Reply:
In all helper functions argument could be a pointer to const struct...

Reply:
done